README.md

```diff
@@ -68,5 +67,5 @@ You can reference the [CONTRIBUTING.md](CONTRIBUTING.md) for a full description
 but the short of it is below.
 
 - If you've found a bug (use the bug label) or want to request a new feature (use the enhancement label), file a GitHub issue
-- If you're not sure about it or want to chat, reach out on our [mailing list](https://groups.google.com/a/lists.stargate.io/g/stargate-users)
+- If you're not sure about it or want to chat, reach out on our [Discord](https://discord.gg/nsE6TJx) or [mailing list](https://groups.google.com/a/lists.stargate.io/g/stargate-users)
```

Comment:
FYI @denisekgosnell

Reply:
@dougwettlaufer I am unable to verify in discord if that link has unlimited uses w/ no expiration.

Can we please change to this one, which is set to have no expiration limit with unlimited uses? https://discord.gg/GravUqY

Reply:
It does but I'll switch to yours